test/performance/README.md

```diff
@@ -1,128 +1,168 @@
-## Stress Test Experiments
+## KS Performance Experiments
 
 
```

Comment:
Before getting into the mechanics of how to run the test, the document should paint the big picture. What is the purpose of this test? Why do we need both this and #2011 ? What is the scenario that this test runs? What will be measured? Explain the data path from sensors to whatever end form this test produces.

Reply:
Ok, I see! The current title is a bit misleading. Hence, I changed the header title of this README to "Workload Benchmark for KubeStellar". The goal of this readme is to explain how to generate the sample workload to conduct performance experiments for Kubestellar. I will write few blog post describing in the experiments in details and results.

Reply:
I think that it is OK to let blog posts carry some of the weight, but I think it would be helpful to give some very basic context setting in this README. Can I test this myself, after using the Getting Started setup instructions or `test/e2e/run-test.sh` to setup an environment?

Reply:
Fixed! Add a prerequisite paragraph that provides the basic context setting.